app/helpers/writable_helper.rb

```diff
@@ -7,69 +7,6 @@ def unread_warning
     tag.a('(New tab)', href: unread_path(@post), class: 'unread-warning', target: '_blank')
   end
```

Comment:
This can still call things from icon_helper, right? (Like unread_path)

(Edit: Answered in Discord - it can!)